watertap/examples/flowsheets/electrodialysis/electrodialysis_1stack.py

```diff
@@ -0,0 +1,326 @@
+    m.fs.separator.split_fraction[0, "inlet_diluate"].fix(0.5)
+
+    # Fix ED unit vars
+    m.fs.EDstack.water_trans_number_membrane["cem"].fix(5.8)
+    m.fs.EDstack.water_trans_number_membrane["aem"].fix(4.3)
+    m.fs.EDstack.water_permeability_membrane["cem"].fix(2.16e-14)
+    m.fs.EDstack.water_permeability_membrane["aem"].fix(1.75e-14)
+    m.fs.EDstack.voltage_applied.fix(5)
+    m.fs.EDstack.electrodes_resistance.fix(0)
+    m.fs.EDstack.cell_pair_num.fix(100)
+    m.fs.EDstack.current_utilization.fix(1)
+    m.fs.EDstack.spacer_thickness.fix(2.7e-4)
+    m.fs.EDstack.membrane_areal_resistance["cem"].fix(1.89e-4)
+    m.fs.EDstack.membrane_areal_resistance["aem"].fix(1.77e-4)
+    m.fs.EDstack.cell_width.fix(0.1)
+    m.fs.EDstack.cell_length.fix(0.79)
+    m.fs.EDstack.membrane_thickness["aem"].fix(1.3e-4)
+    m.fs.EDstack.membrane_thickness["cem"].fix(1.3e-4)
+    m.fs.EDstack.solute_diffusivity_membrane["cem", "Na_+"].fix(1.8e-10)
+    m.fs.EDstack.solute_diffusivity_membrane["aem", "Na_+"].fix(1.25e-10)
+    m.fs.EDstack.solute_diffusivity_membrane["cem", "Cl_-"].fix(1.8e-10)
+    m.fs.EDstack.solute_diffusivity_membrane["aem", "Cl_-"].fix(1.25e-10)
+    m.fs.EDstack.ion_trans_number_membrane["cem", "Na_+"].fix(1)
+    m.fs.EDstack.ion_trans_number_membrane["aem", "Na_+"].fix(0)
+    m.fs.EDstack.ion_trans_number_membrane["cem", "Cl_-"].fix(0)
+    m.fs.EDstack.ion_trans_number_membrane["aem", "Cl_-"].fix(1)
+
+    # check degrees of freedom
+    if degrees_of_freedom(m) != 0:
+        raise RuntimeError(
+            "The set_operating_conditions function resulted in {} "
+            "degrees of freedom rather than 0. This error suggests "
+            "that too many or not enough variables are fixed for a "
+            "simulation.".format(degrees_of_freedom(m))
+        )
+
+
+def solve(blk, solver=None, tee=False, check_termination=True):
+    if solver is None:
+        solver = get_solver()
+    results = solver.solve(blk, tee=True)
+    if check_termination:
+        assert_optimal_termination(results)
+    return results
+
+
+def initialize_system(m, solver=None):
+
+    # set up solver
+    if solver is None:
+        solver = get_solver()
+    optarg = solver.options
+
+    # populate intitial properties throughout the system
+    m.fs.feed.initialize(optarg=optarg)
+    propagate_state(m.fs.s01)
+    m.fs.separator.initialize(optarg=optarg)
+    propagate_state(m.fs.s02)
+    propagate_state(m.fs.s03)
+    m.fs.EDstack.initialize(optarg=optarg)
+    propagate_state(m.fs.s04)
+    m.fs.product.initialize()
+    propagate_state(m.fs.s05)
+    m.fs.disposal.initialize()
+    m.fs.costing.initialize()
+
+
+def optimize_system(m, solver=None):
+
+    # Below is an example of optimizing the operational voltage and cell pair number (which translates to membrane use)
+    # Define a system with zero dof
+    set_operating_conditions(m)
```

Comment:
The system already had set_operating_conditions called, at least in the workflow in main(). I'd recommend deleting this here.

Reply:
I kept this for now for cases where the main() is not called.  Will check this back in the future.